storage/pebble/procedure/index.go

```diff
@@ -3,14 +3,14 @@ package procedure
 import (
 	"fmt"
 
-	"github.com/dgraph-io/badger/v2"
+	"github.com/cockroachdb/pebble"
 
 	"github.com/onflow/flow-go/model/flow"
-	"github.com/onflow/flow-go/storage/badger/operation"
+	"github.com/onflow/flow-go/storage/pebble/operation"
 )
 
-func InsertIndex(blockID flow.Identifier, index *flow.Index) func(tx *badger.Txn) error {
-	return func(tx *badger.Txn) error {
+func InsertIndex(blockID flow.Identifier, index *flow.Index) func(tx pebble.Writer) error {
```

Comment:
```suggestion
// InsertIndex adds an entry to the map blockID -> Index.  
// Note that the same key (`blockID`) necessitates that the value (`index`) is
// also identical (otherwise, we would have a successful pre-image attack on our
// cryptographic hash function). Therefore, concurrent calls to this function are safe. 
func InsertIndex(blockID flow.Identifier, index *flow.Index) func(tx pebble.Writer) error {
```